src/parse/state/tag.ts

```diff
@@ -132,7 +132,7 @@ export default function tag(parser: Parser) {
 		? meta_tags.get(name)
 		: (/[A-Z]/.test(name[0]) || name === 'svelte:self' || name === 'svelte:component') ? 'InlineComponent'
 		: name === 'title' && parent_is_head(parser.stack) ? 'Title'
-		: name === 'slot' && !parser.customElement ? 'Slot' : 'Element';
+		: name === 'slot' && !parser.shadowDom ? 'Slot' : 'Element';
```

Comment:
Another area I wasn't 100% sure on. From my reading `slot` seems to be directly related to the shadow DOM. I assume that with a custom element we'd just want the `slot` to be a regular element..